newscoop/src/Newscoop/NewscoopBundle/Controller/CommentsController.php

```diff
@@ -298,11 +299,11 @@ public function setRecommendedAction(Request $request, $comments, $recommended)
                 continue;
             }
             $comment = $em->getRepository('Newscoop\Entity\Comment')->find($commentId);
-
+            $thread = $em->getRepository('Newscoop\Entity\Article')->findOneBy(array('number' => $comment->getThread()));
```

Comment:
hmm... thread is required - you can't create comment without article attached, and on removing articles we remove also comments - i tested it so in my opinion - check is not needed - it works like it worked before (before it was lazy loaded based on relation).